backend/app/tests/routers/test_records.py

```diff
@@ -0,0 +1,52 @@
+from fastapi.testclient import TestClient
```

Comment:
A lot going on here. You should rely on pytest's `conftest` and fixtures.

Reply:
Yes, I am using `conftest` and fixtures now.